applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Should we call it `nrf_cloud_codec`? Or is this more than nRF Cloud?

Reply:
It basically is the `nrf_cloud_codec`, with the prefix removed to make it more generic. It wraps data in simple JSON strings so that it can be received and easily parsed by any cloud solution. It could possibly use IPSO objects or something to do it a standardized way.